apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -399,7 +403,17 @@ def shell_path
     # The connection information for this session (job must be running)
     # @return [OpenStruct] connection information
     def connect
-      OpenStruct.new YAML.safe_load(connect_file.read)
+      if native_host?
+        OpenStruct.new info.native
```

Comment:
`host`, `port` and `password` are top level in `script.natvie`.  We could instead use `connection_info: {"host": some_host, "port": 1234 }` if that's preferable.

Reply:
script.connection_info maybe is preferable - fewer keys that might conflict

Reply:
Changed to `info.ood_connection_info` here and awaiting a PR in ood_core for the same.